src/theme/colors.ts

```
@@ -37,4 +39,8 @@ export const Colors = {
   itemBlue: "#005DEC",
   itemPurple: "#9404C4",
   itemPink: "#FF6FBC",
-};
+
+  //Calendar colors
+  cherry: "#C25E5E",
+  oat: "#78706A",
+};
```

ESLint

[error] Insert `⏎`

(prettier/prettier)